packages/libs/eda/src/lib/map/analysis/mapTypes/shared.tsx

```diff
@@ -365,3 +369,23 @@ export const GLOBAL_VIEWPORT = {
     right: 180,
   },
 };
+
+/**
+ * Error handling helpers
+ */
+
+const noDataPatterns = [
+  /did not contain any data/,
+  /Could not generate continuous variable metadata/,
```

Comment:
When does this happen?

Reply:
I'm pretty sure both are caused by completely empty subsets - this is the error the back end throws to prevent the R stuff blowing up.

The first one comes from the `map-markers` endpoint and the second one from `continuous-variable-metadata`

Reply:
Added some comments to help future coders